experimental/framework/benchmark/src/Reporter.ts

```diff
@@ -0,0 +1,323 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/*
+The MIT License (MIT)
+
+Copyright (c) 2015 Robert Klep
```

Comment:
Is this available as a package, not as a source file?
Can it screw attribution? For example, when we did React Native development in Office (native apps), we would generate attribution file (i.e. all references to all copyrights / contributors) and I believe it was based on extracting licenses from packages (i.e. we have "LICENSE" file in each package).
I'm not a layer, but I think LICENSE file in this package should have attribution to the author.

Reply:
This file has been very heavily modified from its original form, so if you found it in its original form, that would not meet our needs anyway. I've rewritten most of it multiple times (ex: to make it much more strongly typed) and much of the original logic has been removed. It is still clearly derived from Robert Klep's original though, so I suspect its still "substantial portions of the Software"

In my prior use of this code, since it was only used as a dev dependency and never distributed other than as the full source, so I believe this setup was fine, however with it going into fluid-framework I'm not sure what the right approach is other than ask the layers.

Maybe there is something easy we can do since this code is MIT like fluid framework: perhaps add Robert Klep to the authors or something?

We have an MS internal repo showing the version history of this file as it diverged from the original: let me know if that would help here.